include/ck/tensor_operation/operator_transform/transform_conv_bwd_data_to_gemm_v1.hpp

```diff
@@ -544,7 +546,7 @@ struct TransformConvBwdDataToGemm_v1
             const auto wei_gemmbk0_gemmn_gemmbk1_grid_desc =
                 ck::tensor_operation::device::PadTensorDescriptor(
                     wei_gemmbk0_gemmnraw_gemmbk1_grid_desc,
-                    make_tuple(BK0, GemmNPerBlock, BK1),
+                    make_tuple(GemmKPerBlock, GemmNPerBlock, BK1),
                     Sequence<false, DoPadGemmN, false>{});
```

Comment:
Here it doesn't take any effect after all, since in `DoPads` paramter you pass `false` for padding K0

Reply:
REmoved, it is for 1x1 specialization so it is not related.